internal/version/version.go

```diff
@@ -0,0 +1,177 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package version
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+)
+
+var (
+	mon = monkit.Package()
+	// Timestamp is the UTC timestamp of the compilation time
+	Timestamp string
```

Comment:
Timestamp, CommitHash, Version, and Release should all be lowercased, not exported, and it turns out must all be strings including release. Only Build should be exported (first letter capital).

I propose we do buildTimestamp, buildCommitHash, buildVersion, and buildRelease, and make sure to update the shell script

Reply:
Let me do that 👍